ghost/collections/src/CollectionsService.ts

```diff
@@ -0,0 +1,34 @@
+import {Collection} from './Collection';
+
+type CollectionsServiceDeps = {
+    repository: any;
+};
+
+export class CollectionsService {
+    repository: any;
+
+    constructor(deps: CollectionsServiceDeps) {
+        this.repository = deps.repository;
+    }
+
+    async save(collection: Collection): Promise<Collection> {
+        return await this.repository.save(collection);
+    }
+
+    async getById(id: string): Promise<Collection | null> {
+        return await this.repository.getById(id);
+    }
+
+    async getAll(): Promise<Collection[]> {
+        return await this.repository.getAll();
+    }
+
+    async destroy(id: string): Promise<void> {
+        const collection = await this.getById(id);
+
+        if (collection) {
```

Comment:
@allouis  this is another bit that is strange to implement on the service layer. I think it would be much better to have the `destroy` method baked into the repository interface instead of leaving it up to a service to a leak knowledge about "deleted" property. Wdyt?